pkgs/applications/office/ledger-autosync/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, lib, python3Packages, fetchurl }:
+
+python3Packages.buildPythonApplication rec {
+  version = "1.0.0";
+  src = fetchurl {
+    url = "https://github.com/egh/ledger-autosync/archive/v${version}.tar.gz";
+    sha256 = "603d768c2b56c0238ee0abc9fb54ffb7cfac7573ffc484db3d94e211e9cf5453";
+  };
+  name = "ledger-autosync";
+  propagatedBuildInputs = with python3Packages ;
+   [ asn1crypto beautifulsoup4 cffi cryptography entrypoints fuzzywuzzy idna jeepney
+     keyring lxml mock nose ofxclient ofxhome ofxparse pbr pycparser secretstorage six ];
+
+  doCheck = false; # Checks require ledger as a python package
```

Comment:
Can you paste the errors from the output?
Also do you mean it needs `ledger` as an importable python module?

Reply:
>Also do you mean it needs ledger as an importable python module?

Yes. 

> Can you paste the errors from the output?

One of several test cases that try to import ledger as a python module (this error is thrown with `ledger` in the `checkInputs` list:
```
======================================================================
ERROR: test_one_settleDate (tests.test_weird_ofx.TestWeirdOfxLedgerPython)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/ledger-autosync-1.0.0/ledgerautosync/ledgerwrap.py", line 203, in __init__
    import ledger
ModuleNotFoundError: No module named 'ledger'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/build/ledger-autosync-1.0.0/tests/test_weird_ofx.py", line 90, in setUp
    self.lgr = LedgerPython(os.path.join('fixtures', 'empty.lgr'))
  File "/build/ledger-autosync-1.0.0/ledgerautosync/ledgerwrap.py", line 205, in __init__
    raise Exception("Ledger python interface not found!")
Exception: Ledger python interface not found!
```

Reply:
Looks like they don't have python3 bindings https://github.com/ledger/ledger/issues/510

Reply:
So is it ok then to just leave the `doCheck = false;` there?

Reply:
yes. Wouldn't hurt adding we can't add `ledger` as python package because python3 bindings don't exist